packages/babel-core/test/config-chain.js

```diff
@@ -533,7 +532,7 @@ describe("buildConfigChain", function() {
           fixture("nonexistant-fake", "other.js"),
         ],
       });
-      assert.equal(opts1, null);
```

Comment:
`toBe(null)` could be `toBeNull()`
also in other places

Reply:
Changed. Thanks for pointing it out.